taskcluster/docker/base-android/install-sdk.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script is also called at each run of the decision task
+
+set -ex
+
+export ANDROID_SDK_SHA256='444e22ce8ca0f67353bda4b85175ed3731cae3ffa695ca18119cbacef1c1bea0'
+export ANDROID_SDK_VERSION='3859397'
+export ANDROID_SDK_ROOT='/builds/worker/android-sdk-linux'
+export GRADLE_OPTS='-Xmx4096m -Dorg.gradle.daemon=false'
+export SDK_ZIP_LOCATION="$HOME/sdk-tools-linux.zip"
+
+: ${CURL:='curl --location --retry 5'}
+
+$CURL --output "$SDK_ZIP_LOCATION" "https://dl.google.com/android/repository/sdk-tools-linux-${ANDROID_SDK_VERSION}.zip"
+echo "$ANDROID_SDK_SHA256  $SDK_ZIP_LOCATION" | sha256sum --check
```

Comment:
Before then, each decision task run didn't check neither the integrity nor the origin of the SDK. Now, we have the former. I haven't found a way to verify the later.